Main.py

```diff
@@ -108,37 +105,41 @@ def try_get(dictionary, key, lower = False, typer = lambda x: x, default = None)
             if Config.timer_msg_broadcast > 0:
                     schedule.every(Config.timer_msg_broadcast).seconds.do(BOT.broadcastMessage)
             """
-            BOT.broadcastMessage() # Send initial message into channel
+            BOT.broadcastMessage()  # Send initial message into channel
 
             # debug
-            """
-            BOT.setResetroster(ts3conn, "2020-04-01", red = ["the name with the looooong name"], green = ["another really well hung name", "len", "oof. tat really a long one duuuude"], blue = ["[DUST] dude", "[DUST] anotherone", "[DUST] thecrusty dusty mucky man"], ebg = [])
-            """
-            testguilds = [("Unleash Chaos", "uC", "uC")
-                        , ("Requiem of Execution", "RoE", "RoE")
-                        , ("Zum Henker", "ZH", "ZH")
-                        , ("Formation Wolke", "Zerg", "Zerg.")
-                        , ("Zergs Rebellion", "Zerg", "Zerg")
-                        , ("Flussufer Beach Boys", "FBB", "FBB")
-                        , ("Ups Falsche Taste", "UPS", "UPS")
-                        , ("Rising River", "Side", "Side")
-                        , ("Demons of Dawn", "DoD", "DoD")]
-
-            #for gname, gtag, ggroup in testguilds:
+            # BOT.setResetroster(ts3conn, "2020-04-01",
+            #                    red = ["the name with the looooong name"],
+            #                    green = ["another really well hung name", "len", "oof. tat really a long one duuuude"],
+            #                    blue = ["[DUST] dude", "[DUST] anotherone", "[DUST] thecrusty dusty mucky man"],
+            #                    ebg = [])
+            testguilds = [
```

Comment:
Rather a todo, but this should obviously rather be in a separate test file, now that we are introducing actual unit tests.